### ISSUE TITLE
Local File Transfer

### ISSUE THREAD

**Issue by MisterAlanBradley** [strongbox-password-safe/Strongbox#212]
Going through Strongbox the only thing stopping me from entirely switching is a function to locally transfer databse files. If you have a look at Keepass Touch for example. I am able to open a sync via my local wifi. Just enable it in Keepass Touch, than go to the IP of the device. I can either drag'n'drop the database file or select and upload it. An existing database with the same name and password gets updated without having to add the password or faceID again.

Would be something that would be a hugh + for that app. Especially if you dont want to sync via the internet or using iTunes to transfer the files. Maybe you consider to add something like that :)

**Comment by mmcguill** [comment]
Hi @MisterAlanBradley - How about using AirDrop? This works well for people and I use it myself. Super easy to do. Just turn on AirDrop, drop your database to your device and choose 'Open with Strongbox'

**Comment by MisterAlanBradley** [comment]
That would indeed be an option if you want to sync between a Mac and an iOS device. 

If you are using Windows, Linux or an additional Android device AirDrop is simply not working. There would a WiFi transfer be an option.

**Comment by mmcguill** [comment]
I see. Yes, that's something I hadn't thought of. Yes, so that seems like it's fairly doable from a quick bit of research... Let's use this issue to track! :)

**Comment by mmcguill** [comment]
This was a super quick one to implement, so I went for it. Available now in 1.43.2. Tap '+', select 'Add Existing' and choose 'Transfer over Local Network'. Will leave issue here open for a few days for comment

**Comment by Slummi** [comment]
@mmcguil - You were incredibly fast with implementing this!
One small proposal for improvment. Display a short note when WLAN is not available (no IP address), instead of an empty URL string.

By the way, I haven't found all the strings for this feature on babel / Crowdin yet. Maybe there's a delay in synchronisation.

**Comment by mmcguill** [comment]
Good idea! Hadn't tested or thought of that scenario. Let's keep this open to track.

Strings should be there for you but I see that they are not... I'll look into this and get back to you

**Comment by MisterAlanBradley** [comment]
Great work. Thank you for the fast implementation!

**Comment by mlfreeman2** [comment]
Does this work with KeePass for Windows?

It claims to support FTP, HTTP, and WebDAV.
https://keepass.info/help/v2/ioconnect.html

**Comment by MisterAlanBradley** [comment]
Yes it’s working with every system you are locally connected. F. e. WiFi. The app opens a webpage where you can upload your database file.
Am 12. Okt. 2019, 18:40 +0200 schrieb Michael Freeman <notifications@github.com>:
